docs/changes/wcs/15795.bugfix.rst

```diff
@@ -0,0 +1,6 @@
+Updated bundled WCSLIB version to 8.2.2. This update fixes character buffer
+overflows in the comment string for the longitude and latitude axes triggered
+by some projections in ``wcshdo()``, and also the formatting for generic
+coordinate systems. For a full list of changes - see
+http://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES or
+`astropy/cextern/wcslib/CHANGES <https://github.com/astropy/astropy/blob/205bb618937df5520be3f9dbee9d31f7f97170da/cextern/wcslib/CHANGES>`_
```

Comment:
Just don't try to link it. You must have done a similar link when you last updated WCSLIB so it already exists somewhere in the change log because it contains changes from older releases as well.

```suggestion
``astropy/cextern/wcslib/CHANGES``.
```

Reply:
I get the same warnings locally as RTD using your suggestion. That link is not the issue.

Reply:
RTD is fine on main and other PRs, so this warning has to be from your diff somewhere. And the warning specifically point to `docs/changelog.rst`. 🤷‍♀️

Reply:
Regardless, we usually don't do live links in the change log because when links break in the future, we have to "change history" when we have to fix the change log. So I think you should remove the linking regardless.

Reply:
well, let's see.